### PR TITLE
HelpTrigger in header

### PR DESCRIPTION
**Pull request by ranbena**
New help icon in header leading to help homepage.

screen shot 2019-02-18 at 16 32 36
----------------
### Technical notes:
* Swapped AntD icon to FA to match already existing help in SaaS version. Looks like this in other locations:
screen shot 2019-02-18 at 17 03 22
*  Since HelpTrigger renders in Angular as with class wrapper element, had to add temporary css rules to compensate.

### CODE REVIEW

File: client/app/components/HelpTrigger.jsx [hunk 1]
```diff
@@ -1,32 +1,42 @@
+
```
Comment (kravets-levko) [inline client/app/components/HelpTrigger.jsx]: Please remove this empty line

File: client/app/assets/less/redash/redash-newstyle.less [hunk 2]
```diff
@@ -669,6 +669,7 @@ body {
   }
 }
 
+.navbar-link-ANGULAR_REMOVE_ME,
```
Comment (kravets-levko) [inline client/app/assets/less/redash/redash-newstyle.less]: Very interesting use of `ANGULAR_REMOVE_ME` :slightly_smiling_face:

**Comment by ranbena** [comment]
@arikfr you cool to land this?

File: client/app/components/HelpTrigger.jsx [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
 import './HelpTrigger.less';
 
 const DOMAIN = 'https://redash.io';
-const HELP_PATH = '/help/user-guide/';
+const HELP_PATH = '/help';
 const IFRAME_TIMEOUT = 5000;
 const TYPES = {
+  HOME: [
+    '',
+    'Redash Knowledge Base',
```
Comment (arikfr) [inline client/app/components/HelpTrigger.jsx]: Maybe just `Help`? Will be more fitting on the SaaS side.

Comment (ranbena) [reply client/app/components/HelpTrigger.jsx]: Done

**Pull request merged**